package.json

```diff
@@ -99,24 +99,31 @@
     "@cerner/terra-docs": "^1.0.0",
     "classnames": "^2.2.5",
     "eventemitter3": "^4.0.4",
+    "keycode-js": "^2.0.1",
```

Comment:
can we go with keycode-js v3?

Reply:
Only thing keeping me from using 3 is our usage of 2 throughout our other components. Updating would throw some duplicate package warnings, not a huge deal?

Reply:
I've started upgrading components to 3. There are some terra-clinical components on 3 and terra-dev-site v7 will be on 3